velox/docs/functions/spark/window.rst

```diff
@@ -0,0 +1,14 @@
+================
+Window functions
+================
+
+Velox window functions can be used to compute SQL window functions.
```

Comment:
Should this be Spark window functions instead ?

Reply:
Updated, thanks.